docs/usage/configuration-options.md

```diff
@@ -2125,12 +2125,13 @@ We recommend that you use the `strict` mode, and enable the `dependencyDashboard
 
 ## keepUpdatedLabel
```

Comment:
On a related note, the `description` for this config option is hard to understand also:

> Label to request Renovate bot always rebase to keep branch updated.

Reply:
"If set, users can add this label to PRs to request they be kept updated with the base branch." ?

Reply:
Do you want to make the change in this PR?